src/handlers/thirdparty.ts

```diff
@@ -220,3 +250,56 @@ export function handleItemReviewed(event: ItemReviewed): void {
 
   item.save()
 }
+
+let currentReceiptId = 0
+
+export function handleItemSlotsConsumed(event: ItemSlotsConsumed): void {
+  // Update Third Party
+
+  const thirdPartyId = event.params._thirdPartyId
+
+  const thirdParty = ThirdParty.load(thirdPartyId)
+
+  if (thirdParty == null) {
+    log.error(
+      'Tried to consume slots for unregistered third party with id {}',
+      [thirdPartyId]
+    )
+    return
+  }
+
+  thirdParty.consumedSlots = thirdParty.consumedSlots.plus(event.params._qty)
+
+  thirdParty.save()
+
+  // Update or create Curation
+
+  const curatorAddress = event.params._sender.toHexString()
+
+  let curation = Curation.load(curatorAddress)
```

Comment:
Curations are per curator to track their progress reviewing Items for a third party. (We still need to find a way to distinguish between items from different collections inside a TP)